packages/core/src/description-list/description-list.stories.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { html } from 'lit-html';
+import '@cds/core/card/register.js';
+
+export default {
+  title: 'Stories/Description List',
+  parameters: {
+    options: { showPanel: true },
+  },
+};
+
+/** @website */
+export function unstyledList() {
+  return html`<dl cds-list>
+    <dt>Name</dt>
+    <dd>Description</dd>
+    <dt>Name</dt>
+    <dd>Description</dd>
+  </dl>`;
+}
+
+/** @website */
+export function customSpacingList() {
+  return html`<dl cds-list cds-layout="vertical gap:md">
+    <dt>Name</dt>
+    <dd>Description</dd>
+    <dt>Name</dt>
+    <dd>Description</dd>
+    <dd>Description 2</dd>
+    <dt>Name</dt>
+    <dt>Name 2</dt>
+    <dd>Description</dd>
+  </dl>`;
+}
+
+/** @website */
+export function horizontalList() {
+  return html`<dl cds-list cds-layout="horizontal gap:xl">
```

Comment:
The thing about a horizontal layout is that I would expect it to be like the horizontal layout of forms. Such that the `dt` and the `dd` would be left and right, instead of stacked on top of one another.

The trick is (as we've seen with forms) setting it such that they all share a common width for the left (key). 

Maybe @coryrylan would have some ideas on how this could be accomplished with a custom property or two. Maybe a shared `--horizontal-dd-width` that can be set? I'm not sure.

I'm also happy to ditch the horizontal definition list for now too. It's possible that we could add it later if needed.

Reply:
When I spoke with Scott about it the other day on slack, the horizontal was basically extra credit. Not my call, but I don't think it's necessary to support the horizontal layout you describe - also because it would probably break down if someone had more than one `dd` or `dt` next to each other.